### DISCUSSION TITLE
Exclude list of accounts

### DISCUSSION THREAD

**Discussion by RichardBachelier** [ltsp/ltsp#831] [General] [answered]
Hello everyone,

I am using LTSP on an Ubuntu 22.04 server with an Ubuntu 22.04 Desktop image.
On this server, I have an LDAP directory with several thousand accounts that grows every day.

I don't want these accounts to appear on the client side.
I have noticed the PWMERGE_SUR instruction, but I am unsure how to create a regular expression that would exclude these LDAP accounts. A little clarification: they all have the same group (GID): 3000.

Is this already possible? Are there any other alternatives?
Thank you for your help.
Best regards.

Richard

**Comment by alkisg** [comment]
No LDAP accounts are **included** in the image at all, so there's nothing to exclude.
If the clients are going to use LDAP, they'll dynamically see all the LDAP accouns, just like the server does.
While if you don't want them to use LDAP, then which user account do you plan on using? Do you want the clients to be kiosk-like machines, with a single account?

**Comment by RichardBachelier** [comment]
Thank you for this very prompt response.
If I understood correctly, the functionality is more or less the same as the previous version for managing these accounts, right? Will the LDAP accounts be recognized natively on the client machines (Our client machines are in fat mode)? Do I need to run 'ltsp initrd' every time a new account is created in LDAP? I may have misunderstood this mechanism, so I would appreciate further clarification.

**Comment by alkisg** [comment, marked as answer]
You won't have to run ltsp initrd when you add or remove user accounts. The workstations will contact LDAP directly.

**Comment by RichardBachelier** [comment]
Yes, I just ran a test again. So, on my client machine, do I not need to install the LDAP client layer?
I also take this opportunity to thank you for your availability and the quality of your software and work.

Best regards.

**Comment by alkisg** [comment]
The LDAP client layer is what makes the LDAP accounts available. You certainly need it in the client image.
